src/main/java/com/mineinabyss/looty/ecs/components/Init.kt

```diff
@@ -0,0 +1,10 @@
+package com.mineinabyss.looty.ecs.components
+
+import com.mineinabyss.geary.ecs.api.autoscan.AutoscanComponent
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@AutoscanComponent
+@SerialName("looty:init")
```

Comment:
I am unable to use any name with more than 1 word: when I do, I always get "no serializer" exception. Still researching.

Reply:
Make sure you only have one `:` and use underscores instead of spaces. It's very specific about it because the SerialName is used to encode to keys in Minecraft which need those requirements.